src/tests/functional/loading_tests.ts

```diff
@@ -63,6 +63,20 @@ export class LoadingTests extends TurboDriveTestCase {
     const contents = await this.querySelector(frameContents)
     this.assert.equal(await contents.getVisibleText(), "Frames: #frame")
   }
+  
+  async "test reloading a frame reloads the content"() {
+    await this.nextBeat
+
+    await this.clickSelector("#loading-eager summary")
+    await this.nextBeat
+
+    const frameContent = "#loading-eager turbo-frame#frame h2"
+	this.assert.ok(await this.hasSelector(frameContent))
+	// @ts-ignore
+    await this.remote.execute(() => document.querySelector("#loading-eager turbo-frame")?.reload())
+	this.assert.ok(await this.hasSelector(frameContent))
```

Comment:
Looks like we're using tabs for some of these lines, can you switch to just spaces?